### DISCUSSION TITLE
Adding a new graph classification dataset

### DISCUSSION THREAD

**Discussion by YingtongDou** [pyg-team/pytorch_geometric#2508] [Show and tell]
Hi, 

I recently curated [a graph classification dataset](https://github.com/safe-graph/GNN-FakeNews) based on PyG dataset API which includes fake/real news propagation graphs on Twitter. Each news propagation graph is a hierarchical tree-structured graph where the root node represents the news, the leaf nodes are Twitter users who retweeted the root news. The [paper](https://arxiv.org/pdf/2104.12259.pdf) introducing the dataset is accepted by SIGIR'21.

I would like to know if this new dataset can be added to `torch_geometric.datasets` since the tree-structured graph is unique and is extracted from real-world data.

**Comment by rusty1s** [comment]
Sure,  nice work! Feel free to send a PR.
